transport/src/main/java/io/netty/channel/SingleThreadIoEventLoop.java

```diff
@@ -34,10 +35,10 @@
 public class SingleThreadIoEventLoop extends SingleThreadEventLoop implements IoEventLoop {
 
     // TODO: Is this a sensible default ?
-    protected static final int DEFAULT_MAX_TASKS_PER_RUN = Math.max(1,
-            SystemPropertyUtil.getInt("io.netty.eventLoop.maxTaskPerRun", 1024 * 4));
+    protected static final long DEFAULT_MAX_TASKS_PER_RUN_NS = TimeUnit.MILLISECONDS.toNanos(Math.max(10,
```

Comment:
Let's use 100 ms which is the default CFS period (see https://docs.kernel.org/scheduler/sched-bwc.html)

Reply:
@franz1981 Do you want to use 100 ms for the minimum quantum, or for the maximum? When I first saw your comment I thought you meant maximum quantum, so as to improve responsiveness in case we get more I/O work while working on the tasks, but on second reading I realize it's not fully clear which one you mean.